app/views/notification_mailer/very_overdue_notification.text.erb

```diff
@@ -1,9 +1,16 @@
 <%= raw @info_request.public_body.name %> <%= _('are long overdue.')%>
 
+<% if @info_request.public_body.not_subject_to_law? %>
+<%= _('They have still not replied to your {{law_used_short}} request ' \
```

Comment:
I think we can remove the conditionals on this and use "normally required by law" for both situations.